woocommerce/woocommerce
# Fix e2e product import

### Changes proposed in this Pull Request:

This is a followup PR to #30977. It eliminates the hardcoded path to `sample_products.csv` which would only be valid in this repository. It adds a build command that copies the file into the package to ensure it will be found in the core tests package.

### How to test the changes in this Pull Request:

1. Run `cd plugins/woocommerce`
2. Run `pnpm install`
3. Run `pnpx wc-e2e docker:up`
4. Run `pnpx wc-e2e test:e2e`
5. All `Import Products from a CSV file` tests should pass
